File: laravel-demo/app/Http/Controllers/FormsController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class FormsController extends Controller
{
    public function controls()
    {
        return view('forms.controls', ['title' => 'Controls - Forms']);
    }

    public function customControls()
    {
        return view('forms.custom-controls', ['title' => 'Custom controls - Forms']);
    }

    public function dualListbox()
    {
        return view('forms.dual-listbox', ['title' => 'Bootstrap Dual Listbox - Forms']);
    }

    public function editors()
    {
        return view('forms.editors', ['title' => 'Editors - Forms']);
    }

    public function extras()
    {
        return view('forms.extras', ['title' => 'Extras - Forms']);
    }

    public function fileUpload()
    {
        return view('forms.file-upload', ['title' => 'File upload - Forms']);
    }

    public function inputGroups()
    {
        return view('forms.input-groups', ['title' => 'Input groups - Forms']);
    }

    public function layouts()
    {
        return view('forms.layouts', ['title' => 'Layouts and elements - Forms']);
    }

    public function pickers()
    {
        return view('forms.pickers', ['title' => 'Pickers - Forms']);
    }

    public function selects()
    {
        return view('forms.selects', ['title' => 'Selects and tags - Forms']);
    }

    public function sliders()
    {
        return view('forms.sliders', ['title' => 'Sliders - Forms']);
    }

    public function switchers()
    {
        return view('forms.switchers', ['title' => 'Switchers - Forms']);
    }

    public function typeahead()
    {
        return view('forms.typeahead', ['title' => 'Typeahead - Forms']);
    }

    public function validation()
    {
        return view('forms.validation', ['title' => 'jQuery Validation - Forms']);
    }

    public function wizard()
    {
        return view('forms.wizard', ['title' => 'SmartWizard - Forms']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/FormsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers
{
    public class FormsController : Controller
    {

         public ActionResult Controls()
        {
            return View("~/Views/Forms/Controls.cshtml");
        }

         public ActionResult CustomControls()
        {
            return View("~/Views/Forms/CustomControls.cshtml");
        }

         public ActionResult DualListbox()
        {
            return View("~/Views/Forms/DualListbox.cshtml");
        }

         public ActionResult Editors()
        {
            return View("~/Views/Forms/Editors.cshtml");
        }

         public ActionResult Extras()
        {
            return View("~/Views/Forms/Extras.cshtml");
        }

         public ActionResult FileUpload()
        {
            return View("~/Views/Forms/FileUpload.cshtml");
        }

         public ActionResult InputGroups()
        {
            return View("~/Views/Forms/InputGroups.cshtml");
        }

         public ActionResult Layouts()
        {
            return View("~/Views/Forms/Layouts.cshtml");
        }

         public ActionResult Pickers()
        {
            return View("~/Views/Forms/Pickers.cshtml");
        }

         public ActionResult Selects()
        {
            return View("~/Views/Forms/Selects.cshtml");
        }

         public ActionResult Sliders()
        {
            return View("~/Views/Forms/Sliders.cshtml");
        }

         public ActionResult Switchers()
        {
            return View("~/Views/Forms/Switchers.cshtml");
        }

         public ActionResult Typeahead()
        {
            return View("~/Views/Forms/Typeahead.cshtml");
        }

         public ActionResult Validation()
        {
            return View("~/Views/Forms/Validation.cshtml");
        }

         public ActionResult Wizard()
        {
            return View("~/Views/Forms/Wizard.cshtml");
        }

    }
}
